package.json

```diff
@@ -49,7 +49,7 @@
     "split-e2e-tests": "yarn ts-node ./scripts/split-e2e-tests.ts && git add .circleci/config.yml",
     "test-changed": "lerna run test --since dev",
     "test-ci": "lerna run test --concurrency 4 -- --ci -i",
-    "test": "lerna run test",
+    "test": "lerna run test --concurrency 1",
```

Comment:
why concurrency 1 ?

Reply:
it crashes local machine otherwise, tried on 2 different machines, I can poke around with other values but by default it was "blue screening" every time